runner/app/routes/llm_generate.py

```diff
@@ -0,0 +1,123 @@
+import logging
+import os
+from typing import Annotated, Optional, List
+from fastapi import APIRouter, Depends, Form, status, Request
+from fastapi.responses import JSONResponse, StreamingResponse
+from fastapi.security import HTTPAuthorizationCredentials, HTTPBearer
+from app.dependencies import get_pipeline
+from app.pipelines.base import Pipeline
+from app.routes.util import HTTPError, LlmResponse, TextResponse, http_error
+import json
+
+router = APIRouter()
+
+logger = logging.getLogger(__name__)
+
+RESPONSES = {
+    status.HTTP_200_OK: {"model": LlmResponse},
+    status.HTTP_400_BAD_REQUEST: {"model": HTTPError},
+    status.HTTP_401_UNAUTHORIZED: {"model": HTTPError},
+    status.HTTP_500_INTERNAL_SERVER_ERROR: {"model": HTTPError},
+}
+
+
+@router.post(
+    "/llm-generate",
+    response_model=LlmResponse,
+    responses=RESPONSES,
+    description="Generate text responses from input prompts using a large language model.",
+    operation_id="genLlm",
+    summary="LLM Generate",
+    tags=["generate"],
+    openapi_extra={"x-speakeasy-name-override": "llm"},
+)
```

Comment:
I would remove `generate` from the endpoint as it gets redundant, since we are already calling all the AI APIs "generate" for now. My current preferred option would be:
```suggestion
@router.post(
    "/chat-completion",
    response_model=LlmResponse,
    responses=RESPONSES,
    description="Generate text responses from input prompts using a language model.",
    operation_id="genChatCompletion",
    summary="Chat Completion",
    tags=["generate"],
    openapi_extra={"x-speakeasy-name-override": "chatCompletion"},
)
```

In the SDK, this would look like: `client.generate.chat_completion(...)`

WDYT?

Reply:
Update, @kyriediculous made a good point about not using "chat completion" to avoid confusion since we don't really implement OpenAIs interface. My current preferred options are now:
- `/chat`
- `/text`
- `/llm`

With the corresponding operation_id/summary/speakeasy name.

Reply:
Thanks for your suggestions. My preference is with `Chat` as `GenerateChat` does also make sense 👍.